tests/testthat/test-mmrm-methods.R

```diff
@@ -140,3 +140,39 @@ test_that("confint give same result as emmeans if no ", {
   expect_equal(conf$lower.CL, conf_coef[, 1])
   expect_equal(conf$upper.CL, conf_coef[, 2])
 })
+
+test_that("confint give same result as SAS on unstructured", {
+  object <- mmrm(FEV1 ~ ARMCD + SEX + us(AVISIT | USUBJID), data = fev_data, reml = FALSE)
```

Comment:
would be good to have a comment in here and below test to refer to which design/SAS script was used / which output file the numbers come from